mrp_production_capacity/models/mrp.py

```diff
@@ -38,10 +24,14 @@ class MrpRoutingWorkcenter(models.Model):
 class MrpProduction(models.Model):
     _inherit = 'mrp.production'
 
+    show_split_button = fields.Boolean('Show split button', default=False)
```

Comment:
Pedro... este campo lo necesito a nivel de OF porque lo necesito para saber si tengo que mostrar el nuevo botón "Split quantity".